clientd3d/server.c

```diff
@@ -1650,8 +1651,24 @@ Bool HandleLoginFailed(char *ptr, long len)
 /********************************************************************/
 Bool HandleEnterGame(char *ptr, long len)
 {
-   if (len != 0)
+   int server_move_interval;
+
+   if (len < 4)
+      return False;
+
+   Extract(&ptr, &server_move_interval, 4);   
+
+   // Sanity check to ensure move interval isn't negative or zero
+   if (server_move_interval <= 0)
```

Comment:
Maybe a more aggressive check?  If the move interval is, say, 1, a single client could flood the server.

Reply:
Yes, of course! I'm thinking 250, or 4 moves per second, at most.